### DISCUSSION TITLE
升级到了8.0版本之后，无法授权设备使用专业版

### DISCUSSION THREAD

**Discussion by davidli59** [getlantern/lantern-forum-cn#545] [General]
升级到了8.0版本之后，无法授权设备使用专业版。
通过别的设备输入代码之后，没有反应；通过邮箱设置，点击也没有反应。
账号：48136945@qq.com

**Comment by MaxMKMax** [comment]
@davidli59 您好. 请您也提供我您的支付单据. 我帮您提交问题. 谢谢.
